Framework/Algorithms/inc/MantidAlgorithms/Stitch1DMany.h

```diff
@@ -94,13 +93,13 @@ class DLLExport Stitch1DMany : public API::Algorithm {
   std::vector<double> m_endOverlaps;
   std::vector<double> m_params;
   std::vector<double> m_scaleFactors;
+  std::vector<double> m_manualScaleFactors;
```

Comment:
Some calls to the above functions will pass their own values in instead of member variables (e.g. doStitch1D() at line 398 passes a variable `periodScaleFactors` instead of the `m_manualScaleFactors`). While I suppose its possible to remove some parameters, it feels more consistent to leave them in.